src/main/java/com/conveyal/gtfs/model/LocationGroupStop.java

```diff
@@ -112,29 +113,26 @@ public Iterator<LocationGroupStop> iterator() {
     }
 
     public String toCsvRow() {
-        return String.join(
-            ",",
+        return GeoJsonUtil.createCSVRow(
             location_group_id,
-            (stop_id != null)
-                ? stop_id.contains(",") ? "\"" + stop_id + "\"" : stop_id
-                : ""
-        ) + System.lineSeparator();
+            (stop_id != null) ? stop_id.contains(",") ? "\"" + stop_id + "\"" : stop_id : ""
+        );
     }
 
     /**
      * Extract the location group stops from file and group by location group id. Multiple rows of location groups with
      * the same location group id will be compressed into a single row with comma separated stop ids. This is to allow
      * for easier CRUD by the DT UI.
-     * <p>
+     * <pre>
```

Comment:
Hmm. "E.g" and "will become" are their own non-preformatted text (paragraph was fine) (4 instances)